test/integration/client-side-encryption/driver.test.ts

```diff
@@ -471,3 +476,89 @@ describe('Range Explicit Encryption with JS native types', function () {
     });
   });
 });
+
+describe('CSOT', function () {
+  describe('Explicit Encryption', function () {
+    describe('#createEncryptedCollection', function () {
+      let keyVaultClient: MongoClient;
+      let internalClient: MongoClient;
+      let clientEncryption: ClientEncryption;
+      const LOCAL_KEY = Buffer.from(
```

Comment:
Can we obtain the LOCAL_KEY from the env?  It is available in all CI environments where we run csfle tests , in the `CSFLE_KMS_PROVIDERS` environment variable.

Reply:
(this applies to all tests)